src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/Int32/SmallInt32FrozenDictionary.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NET7_0_OR_GREATER
```

Comment:
I'd rather we just only include the file when building for an appropriate target, e.g. as is done with https://github.com/dotnet/runtime/blob/7fa0d5b5942f138c362f2753398b4a8d4f71eb73/src/libraries/System.Collections.Immutable/src/System.Collections.Immutable.csproj#L77

Reply:
Done.